aula.cabal

```diff
@@ -70,6 +70,7 @@ library
     , vector ==0.10.12.3
   exposed-modules:
       Action
+      Action.Dummy
```

Comment:
Under `Persistent`, we have the module structure `Action.Implementation.*`, where `*` is any implementation.  I guess the equivalent here would be:

- `Action.Implementation` -> `Action.Implementation.Basic`
- `Action.Dummy` -> `Action.Implementation.Dummy`